packages/core/scripts/umip-3/2_VoteSimulate.js

```diff
@@ -19,9 +19,10 @@ const argv = require("minimist")(process.argv.slice(), { boolean: ["revert"] });
 // Address which holds a lot of UMA tokens to mock a majority vote
 const foundationWallet = "0x7a3A1c2De64f20EB5e916F40D11B01C441b2A8Dc";
 
-const Voting = artifacts.require("Voting");
-const VotingInterfaceTesting = artifacts.require("VotingInterfaceTesting");
```

Comment:
@chrismaree is this OK to remove since we fix Voting to a certain ABI now? Is there any reason to use the ancillary data in this script?

Reply:
nope, that seems like a reasonable change.